acceptance/docker-compose.yaml

```diff
@@ -16,7 +16,7 @@ services:
 
   zitadel:
     user: '${ZITADEL_DEV_UID}'
-    image: '${ZITADEL_IMAGE:-ghcr.io/zitadel/zitadel:v2.71.6}'
+    image: '${ZITADEL_IMAGE:-ghcr.io/zitadel/zitadel:v3.3.0}'
```

Comment:
This needed to be bumped as action-targets are only available on the newer versions